exercises/diffie-hellman/.meta/template.j2

```diff
@@ -0,0 +1,35 @@
+{%- import "generator_macros.j2" as macros with context -%}
+{% set class = exercise | camel_case -%}
+{{ macros.header(["private_key", "public_key", "secret"]) }}
+
+class {{ exercise | camel_case }}Test(unittest.TestCase):
+    {% for case in cases -%}
+    {% set property = case["property"] | to_snake -%}
+    {% set description = case["description"] | to_snake -%}
+    {% set expected = case['expected'] %}
+    {% set input = case['input'] %}
+    {% if property == "private_key_is_in_range" -%}
+    def test_{{ description }}(self):
+        primes = [5, 7, 11, 13, 17, 19, 23, 29, 31, 37, 41, 43, 47]
+        for i in primes:
```

Comment:
It's a very minor thing, but could we change the temp value to **p** here, since each is a prime? It'd also jive more with the comments in the canonical data.

Reply:
Sure thing, I've copied mostly of that from the previous test case but I think your right there. Where there's change to improve something - it should be done.

Reply:
Lol I've changed this anf for some reason tests are failing haha.